src/index.js

```diff
@@ -1,38 +1,90 @@
+// eslint-disable-next-line import/named
 import './style.css';
+import add from './modules/add.js';
+import taskRemaining from './modules/taskFunctions.js';
+import {
+  addButton, taskInput, container, clearChecked,
+} from './modules/taskClass.js';
+import { save, retrieve } from './modules/storage.js';
 
-const tasks = [
-  {
-    description: 'Pay the bills',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Pick up the kids',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Write some code',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Drink coffee',
-    completed: true,
-    index: 3,
-  },
-];
+taskInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    if (taskInput.value === '') {
+      e.preventDefault();
+    } else {
```

Comment:
- [ ] You have done great so far, but there are still some issues you need to fix before moving to the next milestone. :+1: :100: 
- [ ] Just as the previous review pointed out on an error preventing the application from running, You can see from the video below, when I start the server, I get an error pointing to an unknown import in your CSS, the application opens when I comment out the import but the add or remove functionality doesn't run, you could see a lot of errors in the console. I'll suggest you carefully go through your project and rectify these errors. :+1: :smile: 

- - Video Reference:

https://github.com/Dorcas126/To-Do-List/assets/107632780/9c182916-75da-464d-95e9-7f8e1376ae86




- [ ] As required by Microverse, you are not meant to use any linters disable rule in your project. I'll suggest you remove the disable cmd in `line 1` of this file and try to rectify the linters issue. :+1: :100: